storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -53,5 +53,25 @@ include::modules/persistent-storage-csi-vsphere-top-aware.adoc[leveloffset=+1]
 .Additional resources
 * https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-esxi-management/GUID-16422FF7-235B-4A44-92E2-532F6AED0923.html?hWord=N4IghgNiBcIC5gOYgL5A[VMware vSphere tag documenation]
 
+== vSphere encryption
+
+You can encrypt virtual machines (VMs) and persistent volumes (PVs) on {product-title} running on top of vSphere. This feature allows user-provisioned persistent storage to be encrypted over wire. 
+
+// Add xrefs to "installation" and "post-installation".
+
+Encrypting vSphere VMs and PVs is a multi-step process, which can be accomplished during installation or post-installation. The final step in setting up this encryption feature requires that you configure a storage class that supports dynamic volume encryption using the vSphere Container Storage Interface (CSI) driver, which can be accomplished in one of two ways using:
+
+* xref:../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-encryption-datastore-url_persistent-storage-csi-vsphere[Datastore URL]
++
+This approach is not very flexible and forces you to use a single datastore and does not support topology-aware provisioning. 
+
+* xref:../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-encryption-tag-based_persistent-storage-csi-vsphere[Tag-based placement]
++
+Encrypts the provisioned volumes and uses tag-based placement to target specific datastores. 
+
+include::modules/persistent-storage-csi-vsphere-encryption-datastore-url.adoc[leveloffset=+2]
+
+include::modules/persistent-storage-csi-vsphere-encryption-tag-based.adoc[leveloffset=+2]
+
 == Additional resources
```

Comment:
Out of scope, but if you're open to it, this might be a good time to fix the additional resources section (by adding an ID and a role). Source: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies